src/containers/AppWithClusters/ExtendedNode/ExtendedNode.tsx

```diff
@@ -1,10 +0,0 @@
-import {useClusterBaseInfo} from '../../../store/reducers/cluster/cluster';
-import type {Node} from '../../Node/Node';
-import {useAdditionalNodeProps} from '../useClusterData';
-
-export function ExtendedNode({component: NodeComponent}: {component: typeof Node}) {
-    const {balancer} = useClusterBaseInfo();
```

Comment:
This was moved to `useNodeDeveloperUIHref`, there is no need for this wrapper anymore